include/zephyr/devicetree.h

```diff
@@ -843,9 +843,10 @@
 /**
  * @brief Like DT_STRING_TOKEN(), but uppercased.
  *
- * This removes "the quotes and capitalize" from string-valued properties, and
- * converts non-alphanumeric characters to underscores. That can be useful, for
- * example, when programmatically using the value to form a C variable or code.
+ * This removes "the quotes" and capitalizes the value of a
+ * string-valued property, and converts non-alphanumeric characters to
+ * underscores. That can be useful, for example, when programmatically
+ * using the value to form a C variable or code.
  *
```

Comment:
This removes **"the quotes", capitalizes** the value of a string-valued property, and converts non-alphanumeric characters to underscores. **This** can be useful, for example, when programmatically using the value to form a C variable or code.

Small grammar corrections

Reply:
@catch-twenty-two I took another crack at this, PTAL. Thanks.

Reply:
looks good!